<launch>
  <arg name="detection_topic" default="/hummingbird0/track/bounding_box" />
  <arg name="image_topic" default="/hummingbird0/camera/rgb" />
  <arg name="imu_topic" default="/gaze/imu" />
  <!--arg name="gt_topic" default="/"/-->
  <node pkg="jpdaf_tracker" type="jpdaf_tracker" name="jpdaf_tracker" output="screen">
    <remap from="~detection" to="$(arg detection_topic)" />
    <remap from="~image" to="$(arg image_topic)" />
    <remap from="~imu" to="$(arg imu_topic)" />
    <!--remap from="~gt" to="$(arg gt_topic)"/-->
    <param name="pd" value="0.952662721893" />
    <param name="gamma" value="0.5" />
    <param name="false_measurements_density" value="0.00000003034" />
    <param name="beta_0_threshold" value="0.8" />
    <param name="alpha_0_threshold" value="0.98" />
    <param name="max_missed_rate" value="15" />
    <param name="min_acceptance_rate" value="15" />
    <rosparam param="R">[2, 2]</rosparam>
    <rosparam param="T">[50000, 50000]</rosparam>
    <rosparam param="P_0">[20, 8, 20, 8]</rosparam>
    <param name="nb_drones" value="3" />
    <param name="assoc_cost" value="50" />
    <param name="max_update_time_rate" value="0.1" />
    <param name="focal_length" value="564.26" />
    <param name="alpha_cam" value="1.0" />
    <rosparam param="principal_point">[489.72, 267.87]</rosparam>
    <param name="gt_topic_name" value="/vicon" />
    <param name="source_odom_name" value="/Tobii_Glasses/odom" />
    <rosparam param="target_odom_names">["/Dragon_Fly1/odom", "/Dragon_Fly5/odom"]</rosparam>
    <param name="root" value="/home/arpl/jpdaf_output/" />
    <param name="output_file_name" value="jpdaf_output" />
  </node>
</launch>
